docs/api/tooling/gasless-txs.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 4
+sidebar_label: Gasless and Signless Transactions
+---
+
+# Gasless and Signless Transactions
+
+The **EZ-Transactions package** (aka Easy-transactions) simplifies blockchain interactions by enabling gasless and signless transactions. Anyone can use it to integrate into their dApp projects.
+
+With gasless transactions, the dApp developer covers gas fees for their users (web2-like approach), while signless transactions allow automatic signing via temporary sub-accounts. This combination creates a seamless user experience, as they no longer need to pay gas fees or manually sign each transaction. 
+
+Developers can easily integrate these features into their dApps using provided hooks and providers, such as `GaslessTransactionsProvider` and `SignlessTransactionsProvider`. The package also includes UI components for enabling or disabling these features.
+
+For more details, visit the [GitHub page](https://github.com/gear-foundation/dapps/tree/vt-update-ez-transactions/frontend/packages/ez-transactions).
```

Comment:
All these links are from the "vt-update-ez-transactions" branch, so they will need to be updated once the PR is merged.